cypress/e2e/addItem.cy.ts

```diff
@@ -0,0 +1,24 @@
+describe("Todo-List", () => {
+  beforeEach(() => {
+    cy.visit("http://localhost:3000/");
+  });
+
+  it("displays the heading", () => {
+    cy.get('[data-testid="heading"]').should("contain", "Todo-List");
+    cy.get('[data-testid="btn-add"]').should("be.disabled");
+  });
+
+  it("adds a new todo item", () => {
+    cy.get('[data-testid="input-text"]').type("    ", { force: true });
```

Comment:
I don't think we need force here or anywhere else in the entire project. Can you check?